src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -285,7 +285,7 @@ export class InstructorCourseEditPageComponent implements OnInit {
      * typed as such to accomodate for a use case in SearchService.
      */
     return {
-      googleId: i.googleId,
+      googleId: i.googleId ? i.googleId : '',
```

Comment:
Check the front-end also. If the `googleId` is not available, the whole row should not be shown (instead of showing empty input box).